tools/dind/start-resin-supervisor

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
```

Comment:
I just realised, you renamed the calls but not the script itself, is that correct?

Reply:
Haha, good catch!